<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuyTruckWindow</class>
 <widget class="QWidget" name="BuyTruckWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>661</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="13" column="0">
     <widget class="QLabel" name="DowntimeCostsLabel">
      <property name="text">
       <string>Расходы на простой</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QComboBox" name="CargoTypeComboBox"/>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="PurchaseInfoLabel">
      <property name="text">
       <string>Информация о сделке</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="PriceSpinBox">
      <property name="maximum">
       <number>1000000</number>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Регистрационный номер</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="RouteCostsLabel">
      <property name="text">
       <string>Расходы на трассе с грузом</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="RegisterNumberEdit">
      <property name="maxLength">
       <number>6</number>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="CargoTypeLabel">
      <property name="text">
       <string>Тип перевозимого груза</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QSpinBox" name="EmptyCostsSpinBox">
      <property name="maximum">
       <number>100000</number>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="CityComboBox"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="CityLabel">
      <property name="text">
       <string>Город покупки</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="SellerLabel">
      <property name="text">
       <string>Продавец</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="EmptyCostsLabel">
      <property name="text">
       <string>Расходы на трассе порожняком</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QSpinBox" name="RouteCostsSpinBox">
      <property name="maximum">
       <number>100000</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="PriceLabel">
      <property name="text">
       <string>Стоимость</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QSpinBox" name="DowntimeCostsSpinBox">
      <property name="maximum">
       <number>100000</number>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="SellerEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="CapacityLabel">
      <property name="text">
       <string>Грузоподъемность</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDateEdit" name="PurchaseDateEdit"/>
    </item>
    <item row="14" column="0" colspan="2">
     <widget class="QPushButton" name="PurchaseButton">
      <property name="text">
       <string>Купить</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QSpinBox" name="CapacitySpinBox">
      <property name="maximum">
       <number>100000</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="PurchaseDateLabel">
      <property name="text">
       <string>Дата покупки</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="TruckInfoLabel">
      <property name="text">
       <string>Информация о покупаемом грузовике</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0" colspan="2">
     <widget class="QTableView" name="table"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
